tools/kubevirt_list_targets.py

```diff
@@ -0,0 +1,22 @@
+import os
```

Comment:
Can this file be moved into the gitHub_actions repo? What do you think?

Reply:
I think we should keep this here for now. There's nothing else that's going to be using it.